src/lib/components/AvailableApartments.svelte

```diff
@@ -0,0 +1,271 @@
+<script lang="ts">
+	import { onMount } from 'svelte';
+
+	let apartments: Array<any>;
+	apartments = [];
+	const fetchApartments = () => {
+		fetch('https://api.uruggo.com/listings?search&page=1&limit=10')
```

Comment:
@n3-rd you shouldn't hard code the API endpoint, instead create an environment variable to use

Reply:
BTW use Axios and then create a custom client with a base URL

Something like this
```ts
// lib/client.ts

const client = axios.create({
  baseURL: ...,
  withCredentials: true,
})


export default client
```

Reply:
Sure, Will fix